src/resources/ha-sortable-style-ha-clickable.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from "lit-element";
+
```

Comment:
This file is just a copy/paste of `ha-sortable-style.ts` replacing `mwc-list-item` with `ha-clickable-list-item`.

Reply:
Remove the old unused ones from `ha-sortable-style.ts`.

Reply:
But I think it's still being used by hui-entity-editor and hui-entities-card-row-editor

Reply:
Yeah I guess, we should clean it anyway. It should only contain the shared styles, not the element specific.

Reply:
Okay. Not sure if this is what you wanted, but I've consolidated the two style files into one.